Improve Contributors Section in Release Tiddler

kookma opened the pull request:
This PR addresses the issue https://github.com/Jermolene/TiddlyWiki5/issues/6784
for a contributor section to looks more professional 

I tried to use the smallest CSS and do not touch the macro as much as possible.
For example I did not added a div, but used `a` as a block element.

Jermolene reviewed (commented):
Thanks @kookma it looks great. There are a few minor style issues but otherwise looks good to go.

kookma commented on editions/tw5.com/tiddlers/system/doc-macros.tid:
```diff
@@ -172,13 +172,11 @@ $credit$
 \end
 
 \define .contributors(usernames)
-<ol class="doc-github-contributors">
```
I was not aware of this and wait for @ndarilek review comment!
But I think we can keep them. I go with `<ol>/<li>`

kookma commented on editions/tw5.com/tiddlers/system/doc-styles.tid:
```diff
@@ -252,19 +252,31 @@ a.doc-deprecated-version.tc-tiddlylink {
-
-.doc-github-contributors {
-	list-style: none;
+.doc-github-contributors{
```
I correct this!

kookma commented:
@Jermolene 
I implemented your comments. I reverted the section into `<ol><li>`. Please have a look and if you think CSS can be improved please let me know.
